### ISSUE TITLE
4 or more SSH public keys configured in the user profile make deployments fail on EC2.

### ISSUE THREAD

**Issue by rbf** [slipstream/SlipStreamServer#372]
If 4 or more SSH public keys are configured in the user profile, deployments (at least on `ec2-eu-west`) fail with error:

```
Cannot import the public key. Reason: EC2ResponseError: 400 Bad Request InvalidParameterValueValue (c3NoLXJzYSBBQ...
...9zaC5sb2NhbA==) for parameter PublicKeyMaterial is invalid. Length exceeds maximum of 2048.0bfb622e-eff3-4a6e-bb56-e9c37292c91f
```

https://nuv.la/run/9b78ffae-38c2-49ba-820f-45cecab6e57f

At the moment of this deployment, the user `test` had 4 SSH public keys configured in its profile with a total length of 1571 chars (before encoding to Base64).

**Comment by mebster** [comment]
Further, it looks like EC2 now verifies more strictly the public key.  It looks like the '==' code is required between the key and the description (last optional part).

**Comment by schaubl** [comment]
Fixed in SixSq/SlipStreamConnector-EC2#22
